tests/log.rs

```diff
@@ -26,7 +26,7 @@ async fn test_server_listen(logger: &mut logtest::Logger) {
         .unwrap();
     assert_eq!(
         record.args(),
-        format!("Server listening on http://[::1]:{}", port)
+        format!("Server listening on http://127.0.0.1:{}", port)
```

Comment:
I changed these tests because they kept failing on WSL. "localhost" parse down to "127.0.0.1" instead of the Ipv6 equivalent. Didn't know that was a thing. Changed them so they parse the same on all platforms.